### PR TITLE
UI: File download

### PR DESCRIPTION
**Pull request by deetz99**
issues: #166 https://github.com/bcgov/business-ar/issues/224

- add file download to final screen
- have 2 cards for final screen content instead of 1
- update types
- i18n translations
- new alert category and message
- download function in annual report store
- test new download function 

Screenshot 2024-06-21 at 11 00 02 AM

### CODE REVIEW

File: web/site/components/content/DocumentDownload.vue
Comment (deetz99) [inline web/site/components/content/DocumentDownload.vue:1]: File download buttons to use in md file. Only visible if documents in store filing object

File: web/site/content/en-CA/submitted/platform-info.md
Comment (deetz99) [inline web/site/content/en-CA/submitted/platform-info.md:1]: separate submitted md content into 2 files

File: web/site/content/fr-CA/submitted/success-text.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Thank you for keeping your business up to date. You have successfully filed your BC Annual Report on the Business Registry. 
+A receipt has been emailed to :BusinessEmail.
+
+:DocumentDownload
```
Comment (deetz99) [inline web/site/content/fr-CA/submitted/success-text.md:4]: DocumentDownload md component and new file for submitted md content

File: web/site/pages/submitted.vue
Comment (deetz99) [inline web/site/pages/submitted.vue:1]: use 2 cards for the 2 new md files

add alert component if download fails or 500 error

File: web/site/stores/business.ts [hunk 2]
```diff
@@ -85,6 +85,7 @@ export const useBusinessStore = defineStore('bar-sbc-business-store', () => {
 
     if (response) {
       payStatus.value = response.filing.header.status
+      arStore.arFiling = response
```
Comment (deetz99) [inline web/site/stores/business.ts:88]: need to update arFiling object in store so documents are updated to use inside the DocumentDownload component

File: web/site/stores/business.ts [hunk 3]
```diff
@@ -120,7 +121,7 @@ export const useBusinessStore = defineStore('bar-sbc-business-store', () => {
           throw new Error('Access Denied: Your account does not have permission to complete this task.')
         }
         // assignBusinessStoreValues(taskValue.filing.business)
-        arStore.arFiling = { filing: { header: taskValue.filing.header, annualReport: taskValue.filing.annualReport } }
+        arStore.arFiling = { filing: { header: taskValue.filing.header, annualReport: taskValue.filing.annualReport, documents: taskValue.filing.documents } }
```
Comment (deetz99) [inline web/site/stores/business.ts:124]: update to match new type

File: web/site/tests/unit/stores/annual-report.test.ts [hunk 4]
```diff
@@ -88,4 +88,57 @@ describe('Annual Report Store Tests', () => {
 
     expect(Object.keys(arStore.arFiling).length).toBe(0)
   })
+
+  describe('handleDocumentDownload', () => {
```
Comment (deetz99) [inline web/site/tests/unit/stores/annual-report.test.ts:92]: test new function

**Review by thorwolpert — APPROVED** [review]

**Pull request merged**